src/Matrix.jl

```diff
@@ -2593,6 +2615,32 @@ function minors_iterator(M::MatElem, k::Int)
   return (det(M[rows, cols]) for rows in row_indices for cols in col_indices)
 end
 
+@doc raw"""
+    minors_iterator_with_position(A::MatElem, k::Int)
+
+Return an iterator that computes the `k`-minors of `A` also specifying the row and column indices of the minor.
+
+# Examples
+
+```jldoctest
+julia> A = ZZ[1 2 3; 4 5 6]
+[1   2   3]
+[4   5   6]
+
+julia> first(minors_iterator_with_position(A, 2))
+3-element Vector{Any}:
+ -3
+ [1, 2]
+ [1, 2]
+
+```
+"""
+function minors_iterator_with_position(M::MatElem, k::Int)
+    row_indices = combinations(nrows(M), k)
+    col_indices = combinations(ncols(M), k)
+  return ([det(M[rows, cols]), rows, cols] for rows in row_indices for cols in col_indices)
```

Comment:
```suggestion
  return ((det(M[rows, cols]), rows, cols) for rows in row_indices for cols in col_indices)
```
since the entries of the iterator are inhomogeneous (scalar, vector, vector), it is better to use a 3-tuple instead of a vector for that. This should also help julia infer the entry type (cf. the `Any` in line 2631)

Reply:
Thank you!